### PR TITLE
Re-use order id for capture

### PR DESCRIPTION
**Pull request by daliwei**
cc @pierre for review.

### CODE REVIEW

File: spec/orbital/remote/shared_examples_for_payment_flow.rb [hunk 1]
```diff
@@ -87,6 +88,7 @@
       payment_response.status.should eq(:PROCESSED), payment_response.gateway_error
       payment_response.amount.should == partial_capture_amount
       payment_response.transaction_type.should == :PURCHASE
+      payment_response.second_payment_reference_id = initial_order_id
```
Comment (pierre) [inline spec/orbital/remote/shared_examples_for_payment_flow.rb]: `.should ==` ?

**Review by pierre — CHANGES_REQUESTED** [review]

File: spec/orbital/remote/shared_examples_for_payment_flow.rb (hunk 1 above)
Comment (daliwei) [reply spec/orbital/remote/shared_examples_for_payment_flow.rb]: @pierre fixed.

**Comment by pierre** [comment]
v0.1.10 released.

**Comment by daliwei** [comment]
Thank you @pierre

**Pull request merged**